cocos/editor-support/cocostudio/WidgetReader/Sprite3DReader/CSSprite3D_generated.h

```diff
@@ -0,0 +1,61 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
CSSprite3D_generated.h -> CSSprite3D-generated.h
We use `-` to connect multiple words.